mitxpro/settings_test.py

```diff
@@ -96,35 +102,43 @@ def test_db_ssl_enable(self):
         """Verify that we can enable/disable database SSL with a var"""
 
         # Check default state is SSL on
-        with mock.patch.dict("os.environ", REQUIRED_SETTINGS, clear=True):
-            settings_vars = self.reload_settings()
-            self.assertEqual(
-                settings_vars["DATABASES"]["default"]["OPTIONS"], {"sslmode": "require"}
-            )
+        settings_vars = self.patch_settings(REQUIRED_SETTINGS)
+        self.assertEqual(
+            settings_vars["DATABASES"]["default"]["OPTIONS"], {"sslmode": "require"}
+        )
 
         # Check enabling the setting explicitly
-        with mock.patch.dict(
-            "os.environ",
-            {**REQUIRED_SETTINGS, "MITXPRO_DB_DISABLE_SSL": "True"},
-            clear=True,
-        ):
-            settings_vars = self.reload_settings()
-            self.assertEqual(settings_vars["DATABASES"]["default"]["OPTIONS"], {})
+        settings_vars = self.patch_settings(
+            {**REQUIRED_SETTINGS, "MITXPRO_DB_DISABLE_SSL": "True"}
+        )
+        self.assertEqual(settings_vars["DATABASES"]["default"]["OPTIONS"], {})
 
         # Disable it
-        with mock.patch.dict(
-            "os.environ",
-            {**REQUIRED_SETTINGS, "MITXPRO_DB_DISABLE_SSL": "False"},
-            clear=True,
-        ):
-            settings_vars = self.reload_settings()
-            self.assertEqual(
-                settings_vars["DATABASES"]["default"]["OPTIONS"], {"sslmode": "require"}
-            )
+        settings_vars = self.patch_settings(
+            {**REQUIRED_SETTINGS, "MITXPRO_DB_DISABLE_SSL": "False"}
+        )
+        self.assertEqual(
+            settings_vars["DATABASES"]["default"]["OPTIONS"], {"sslmode": "require"}
+        )
 
     @staticmethod
     def test_semantic_version():
         """
         Verify that we have a semantic compatible version.
         """
         semantic_version.Version(settings.VERSION)
+
+    @staticmethod
+    def test_app_json_modified():
+        """
+        generate_app_json should return a dictionary of JSON config for app.json
+        """
+        from mitxpro.envs import generate_app_json
+
+        with open("app.json") as app_json_file:
+            app_json = app_json_file.read()
+
+        generated_app_json = generate_app_json()
+
+        # pytest will print the difference
+        assert app_json == json.dumps(generated_app_json, sort_keys=True, indent=2)
```

Comment:
This might be a case where we'd want to include an assertion description (or whatever you call it), e.g.:

```python
assert app_json == json.dumps(generated_app_json, sort_keys=True, indent=2), "Generated app.json does not match the app.json file. Please use the 'generate_app_json' management command to update app.json"
```